src/Nethermind/Nethermind.BeaconNode.Test/EpochProcessing/ProcessJustificationAndFinalizationTest.cs

```diff
@@ -392,7 +392,7 @@ private void PutCheckpointsInBlockRoots(BeaconChainUtility beaconChainUtility, T
             foreach (var checkpoint in checkpoints)
             {
                 var startSlot = beaconChainUtility.ComputeStartSlotOfEpoch(checkpoint.Epoch);
-                var slotIndex = startSlot % timeParameters.SlotsPerHistoricalRoot;
+                Slot slotIndex = (Slot)(startSlot % timeParameters.SlotsPerHistoricalRoot);
```

Comment:
Is this because you don't have % on your version of Slot ?

Reply:
oh, yeah, let me fix it - I was moving all your operators and may have missed some when copy pasting (our Slot, Epoch, and similar types had barely any differences inside - mainly missing operators and a few other minor things

Reply:
Nooo - it is because of the explicit casting I think
        public static Slot operator %(Slot left, Slot right)
        {
            return new Slot(left.Number % right.Number);
        }

Reply:
I changed SlotsPerHistoricalRoot to uint / ulong as this is not slot but a number